lib/Common/Codex/Utf8Helper.h

```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 #pragma once
+#include "CommonPal.h" // MAXUINT32
```

Comment:
?

Reply:
Utf8Helper.h uses MAXUINT32, which was probably getting defined by accident each time before Utf8Helper.h got included. Including Utf8Helper.h in PlatformAgnostic/Common/Intl.cpp caused this to come up.